packages/markdown/component/readme.md

```diff
@@ -0,0 +1,15 @@
+# `<Markdown />`
```

Comment:
Would be nice to align this README with the other integration READMEs.

Reply:
The MDX integration README in particular is probably a good comp to use (no configuration etc): https://github.com/withastro/astro/blob/main/packages/integrations/mdx/README.md